### PR TITLE
Init ci

### PR DESCRIPTION
**Pull request by younger15**

### CODE REVIEW

**Review by Jazz86 — APPROVED** [review]
LGTM

**Review by ian287913 — APPROVED** [review]
LGTM

File: .github/workflows/ciUnitTest.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (john0312) [inline .github/workflows/ciUnitTest.yml:1]: This comment might not be needed here. Please remove irrelevant comments.
Same below.

File: .github/workflows/ciUnitTest.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: build docker
+        run: docker build -t unittest -f docker/Dockerfile .
+
+      # Runs a set of commands using the runners shell
+      - name: run docker
+        run: docker run --rm -i unittest
```
Comment (john0312) [inline .github/workflows/ciUnitTest.yml]: It would be recommended to run your build and test process here.

**Review by LinRenHong — APPROVED** [review]

File: .github/workflows/ciUnitTest.yml (hunk 1 above)
Comment (younger15) [reply .github/workflows/ciUnitTest.yml:1]: Done

**Pull request merged**